include/pops/treatments.hpp

```diff
@@ -244,6 +256,7 @@ class PesticideTreatment : public BaseTreatment<IntegerRaster, FloatRaster>
         std::vector<IntegerRaster>& exposed_vector,
         IntegerRaster& susceptible,
         IntegerRaster& resistant,
+        IntegerRaster&,
         const std::vector<std::vector<int>>& suitable_cells) override
     {
```

Comment:
We newly have `UNUSED` defined in `utils.hpp`, so you can do:

```cpp
#include "utils.hpp"
...
    IntegerRaster& total_hosts,
...
{
    UNUSED(total_hosts);
```

which makes it clearly documented at least in the code.